static/mainapp/css/read.css

```diff
@@ -0,0 +1,157 @@
+.genre-head,
+.topbook-head {
+  font-family: "Lobster", cursive;
+  font-size: 300%;
+  background-color: #d7722c;
+  margin-top: 0px;
+  color: white;
+}
+
+.card-img {
+    flex-shrink: 0;
+    bottom: -35px;
+    left: 35px;
+    border-radius: 3px;
+    box-shadow: -2px 6px 19px 0px #7f818e;
+    transition: 0.3s ease;
+    max-width: 100%;
+    max-height: auto;
+    margin: 0 ;
+    padding:0;   
+  }
+  .card {
+    transition: 0.5s;
+    cursor: pointer;
+    border-radius: 3px;
+    box-shadow: -2px 6px 19px 0px #7f818e;
+    margin-left: auto;
+    margin-right:auto;  
+  }
+
+  .card-title {
+    font-family: "Sora", sans-serif;
+    font-size: 2rem;
+    color: #d7722c;
+    padding-bottom: 20px;
+  }
+
+  .card-text {
+    font-family: "Pacifico", cursive;
```

Comment:
font-family for card-text : "Courgette", cursive;
We are following this.

Reply:
@maneprajakta Is this pending?

Reply:
@Praful932  Nope